internal/cli/universal_login_customize_test.go

```diff
@@ -1008,3 +1009,308 @@ func TestCheckOriginFunc(t *testing.T) {
 		})
 	}
 }
+
+func TestWebSocketMessage_MarshalJSON(t *testing.T) {
+	var testCases = []struct {
+		name     string
+		input    *webSocketMessage
+		expected string
+	}{
+		{
+			name: "it can marshal a fetch prompt data message",
+			input: &webSocketMessage{
+				Type: "FETCH_PROMPT",
+				Payload: &promptData{
+					Language:   "en",
+					Prompt:     "login",
+					CustomText: map[string]interface{}{"key": "value"},
+				},
+			},
+			expected: `{"type":"FETCH_PROMPT","payload":{"language":"en","prompt":"login","custom_text":{"key":"value"}}}`,
```

Comment:
Can we expect the marshaller to always generate an identical string (with the key/values in the same order)?